### ISSUE TITLE
AAT error: "element does not have a <legend> for index filter fieldsets"

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#1741]
Running Accessibility Checker on the new Component index page throws an error for missing legends. The fieldsets _do have legends_, but they're nested inside wrapper divs and manifest like this

![image](https://user-images.githubusercontent.com/40970507/93267362-31609600-f771-11ea-8abc-611b00235a47.png)
